jsx/Panel.js

```diff
@@ -65,7 +76,11 @@ Panel.propTypes = {
   title: React.PropTypes.string
 };
 Panel.defaultProps = {
+  initCollapsed: false,
   id: 'default-panel',
-  height: '100%',
-  title: 'Selection Filter'
+  height: '100%'
 };
+
+window.Panel = Panel;
```

Comment:
@driusan and @jacobpenny Its actually used [here](https://github.com/aces/Loris/blob/17.1-dev/php/libraries/NDB_Menu_Filter.class.inc#L852). Technically, I could get rid of it in the separate PR and only require when necessary